code/modules/holiday/holidays.dm

```diff
@@ -661,3 +661,102 @@
 /datum/holiday/pride_week/proc/get_floor_tile_color(atom/atom)
 	var/turf/turf = get_turf(atom)
 	return rainbow_colors[(turf.y % rainbow_colors.len) + 1]
+
+/datum/holiday/remembrance_day
+	name = "Remembrance Day"
+	begin_month = NOVEMBER
+	begin_day = 11
+	drone_hat = /obj/item/food/grown/poppy
+
+/datum/holiday/remembrance_day/getStationPrefix()
+	return pick("Peace", "Armistice", "Poppy")
+
+/datum/holiday/victory_day
+	name = "Victory in Europe Day"
+	begin_month = MAY
+	begin_day = 8
+
+/datum/holiday/victory_day/getStationPrefix()
+	return pick("Liberty", "Victory", "Peace")
+
+/datum/holiday/un_day
+	name = "Anniversary of the Foundation of the United Nations"
+	begin_month = OCTOBER
+	begin_day = 24
+
+/datum/holiday/un_day/greet()
+	return "On this day in 1945, the United Nations was founded, laying the foundation for humanity's united government!"
```

Comment:
corn syrup

Reply:
But there is no united human goverment in the lore?

Reply:
> But there is no united human goverment in the lore?

There is, see https://github.com/tgstation/common_core/blob/master/Governments.md